src/app/user/modules/records/components/record-form/record-form.component.ts

```diff
@@ -132,6 +132,9 @@ export class RecordFormComponent implements OnInit {
     });
     if (record.attachments.length !== 0) {
       this.originalAttachments = record.attachments as RecordAttachment[];
+      this.originalAttachments.map( attachment => {
+        attachment.path = attachment.path as Path;
```

Comment:
I mean you can use `originalAttachments: SecuredFile[] = [];` instead of `originalAttachments: RecordAttachment[] = [];`